packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -35,8 +35,15 @@ export const enum RumTrackingType {
 }
 
 export function startRumSessionManager(configuration: RumConfiguration, lifeCycle: LifeCycle): RumSessionManager {
-  const sessionManager = startSessionManager(configuration.cookieOptions, RUM_SESSION_KEY, (rawTrackingType) =>
-    computeSessionState(configuration, rawTrackingType)
+  if (!configuration.sessionStoreStrategyType) {
+    throw new Error('Cannot initialize RUM Session Manager without a storage.')
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏instead of doing this runtime check, what about restricting allowed storage strategy in rum configuration?

Reply:
Non-null assertion for now.